src/commands/userlookup.ts

```diff
@@ -0,0 +1,54 @@
+import { InteractionResponseType, InteractionResponseFlags } from 'discord-interactions';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { params } from '@serverless/cloud';
+import DiscordInteraction from '../classes/DiscordInteraction';
+import CommandOptionType from '../enums/ICommandOptionType';
+import { ICommand } from '../interfaces/ICommand';
+
+export const command: ICommand = {
+  name: 'userlookup',
+  description: 'Get info and stats for any Playbite user!',
+  options: [
+    {
+      name: 'username',
+      description: 'The username to look up',
+      type: CommandOptionType.STRING,
+      required: true
+    },
+  ],
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const interact = async (interaction: DiscordInteraction, _interactionActionOverwrite?: any): Promise<any> => {
+  const username = interaction.getOptionValue('username') as string;
+
+  // Get leaderboard from API
+  const userStats = await fetch(`${params.BASE_API_URL}/users/${username}/stats`);
+  const userJson = await userStats.json();
+  // If leaderboard is empty, return error
+  if (!userJson) {
+    return {
+      type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+      data: {
+        content: 'User not found.',
+      },
+    };
+  }
+
+  return {
+    type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+    data: {
+      embeds: [
+        {
+          title: userJson["displayName"],
+          description: userJson["bio"],

```

Comment:
For accessing the data from `userJson` can you change it from `userJson["displayName"]` to `userJson.displayName`? Same for the other ones, just to maintain the same way that other data is being accessed in other files. 

If you want to be extra safe, you can change it to `userJson?.displayName`. This method is [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), which will return undefined if the object property doesn't exist, instead of throwing an error.

Reply:
Ah ok sorry I'm not great at typescript lol but I'll fix it :)
Do you have any way I could test the code out on my machine before submitting my changes?

Reply:
Awesome, thanks! 
As for testing, it is a little tricky and involves a bit of setup. 
### Account Setup
1. You will need to create a new bot on discord https://discord.com/developers/applications
2. You will need to create an account on https://www.serverless.com/

### Local Setup
1. Install cloud CLI `	npm i -g @serverless/cloud`
2. Once this is installed, run `cloud`, which will probably ask you for credentials. Once you input those, it should spin up a cloud instance and give you a URL to go to. 

### App Config Setup
1.  Using the URL given, in the discord app you created under general information -> Interactions Endpoint URL, add in <URL>/discord 
2. Once the URL is added to discord you will need to setup the following env variables on cloud.serverless.com: `DISCORD_CLIENT_ID`, `DISCORD_BOT_TOKEN`, `DISCORD_PUBLIC_KEY`. These env variables can be found in the discord developer console where you set up your application. 
3. There is one more env variable you will need called `BASE_API_URL`, which is just `https://playbiteapi.azurewebsites.net/api`

Once all of the configuration is finished, you should now add the discord bot you create into a discord server. Once it is added visit the following URL in your browser `<url_from_local_setup>/set-commands`. This will register all the bot commands with discord. 

After all of this, it should be working (I hope). Let me know if you have any questions or run into issues. 

I hope to make some improvements in the future so this is easier to setup/test, but I think it will still involve quite a bit of manual configuration due to how Discord bots work (as far as I can tell).

Reply:
Nice, thanks :D I'll get on those fixes rn!

Reply:
Hmmmm... (when i run `cloud`)
```
{"type":"error","message":"Cannot find package 'discord.js' imported from /tmp/task/index.ts","stack":"Error: Cannot find package 'discord.js' imported from /tmp/task/index.ts
    at new NodeError (node:internal/errors:371:5)
    at packageResolve (node:internal/modules/esm/resolve:932:9)
    at moduleResolve (node:internal/modules/esm/resolve:978:18)
    at defaultResolve (node:internal/modules/esm/resolve:1080:11)
    at rn (file:///opt/nodejs/cloud/loader.mjs:8:2826)
    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:79:40)
    at link (node:internal/modules/esm/module_job:78:36)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)","code":"ERR_MODULE_NOT_FOUND"}
```

Reply:
nvm fixed with `npm install discord.js`